scripts/generate-framework-agreement-counterpart-signature-pages.py

```diff
@@ -34,10 +34,23 @@
 which should be at `digitalmarketplace-credentials/signatures/<name>.png`
 
 Usage:
-    scripts/generate-framework-agreement-counterpart-signature-pages.py <stage> <framework_slug>
-    <path_to_agreements_repo> <output_folder> [<supplier_id_file>]
+    scripts/generate-framework-agreement-counterpart-signature-pages.py [options]
+        [--supplier-id=<id> ... | --supplier-ids-from=<file>]
+        <stage> <framework_slug> <path_to_agreements_repo> <output_folder>
 
+Options:
+    <stage>                     Environment to run script against.
+    <framework>                 Framework slug.
+    <path_to_agreements_repo>   Path to directory with agreements repo.
+    <output_folder>             Path to output PDFs.
+
+    --supplier-id=<id>          ID(s) of supplier(s) to email.
+    --supplier-ids-from=<file>  Path to file containing supplier ID(s), one per line.
+
+    -n, --dry-run               Run script without sending emails.
```

Comment:
Can we leave this off until there actually is a dry-run option?

Reply:
Woops, copy-paste mistake

Reply:
Good catch